app/src/main/java/chat/rocket/android/api/rest/ServerPolicyApi.java

```diff
@@ -0,0 +1,19 @@
+package chat.rocket.android.api.rest;
+
+import android.support.annotation.NonNull;
+import org.json.JSONObject;
+
+public interface ServerPolicyApi {
+
+  void getApiInfoSecurely(@NonNull Callback callback);
+
+  void getApiInfoInsecurely(@NonNull Callback callback);
+
+  interface Callback {
```

Comment:
I think this callback interface would be shared for all REST APIs. So it is better to define isolatedly.